eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/SerializableDoubleSummaryStatistics.java

```diff
@@ -61,7 +63,12 @@
             max = DoubleSummaryStatistics.class.getDeclaredField("max");
             max.setAccessible(true);
         }
-        catch (SecurityException | NoSuchFieldException ignored)
+        catch (RuntimeException e)
```

Comment:
I believe catching RuntimeException is actually a regression. We should have a test that fails due to this code, perhaps using a SecurityManager. We can delete this block entirely and just catch Exception. It's one of those rare situations where catching Exception feels right, because it executes in a static block.